docs/reference/release-notes/7.14.asciidoc

```diff
@@ -3,6 +3,22 @@
 
 Also see <<breaking-changes-7.14,Breaking changes in 7.14>>.
 
+[[known-issues-7.14.0 to 7.14.2]]
```

Comment:
I think this does not work, see the [preview](https://elasticsearch_78728.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/7.14/release-notes-7.14.2.html).

Reply:
Nice, I have always wondered how I can preview doc changes on our side.  Thx for the link!  I will remove the anchor and just have the info inline :)